### DISCUSSION TITLE
Who can add v language plug-in to zed?

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#21520] [General]
https://github.com/zed-industries/zed

zed is very comfortable to use and very fast; I hope there can be a v language plug-in

**Comment by JalonSolov** [comment]
You would have to find someone who programs in Rust, uses zed, and is also interested in V.

**Reply by Jengro777** [reply]
https://github.com/lv37/zed-v

**Reply by Delta456** [reply]
Any questions or the same to use it?

**Reply by Jengro777** [reply]
Very good

**Reply by changrui** [reply]
How to use this extensions? I really need it. Thanks.
